.github/workflows/build.yml

```diff
@@ -22,10 +22,10 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                php: ["8.1"]
-                symfony: ["^5.4", "^6.0"]
-                sylius: ["~1.12.0", "~1.13.0"]
-                node: ["^14.17.x"]
+                php: [ "8.1", "8.2", "8.3" ]
```

Comment:
Sylius 1.12 has PHP ^8.0 rule.

Reply:
We can't use PHP 8.0 because the code has constructors for PHP 8.1.

Reply:
Good. I think we won't rewrite the constructors.